From 532785130c536bc0eb38004c26324fb934b43f8c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Edgar=20Andr=C3=A9s=20Margffoy=20Tuay?= <andfoy@gmail.com>
Date: Thu, 2 Dec 2021 16:13:20 -0500
Subject: [PATCH] Add use_system_webp flag to gn_args

---
 src/buildtools/config/common.pri | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/buildtools/config/common.pri b/src/buildtools/config/common.pri
index cf990c79..e099349b 100644
--- a/src/buildtools/config/common.pri
+++ b/src/buildtools/config/common.pri
@@ -30,7 +30,8 @@ gn_args += \
     use_allocator=\"none\" \
     use_custom_libcxx=false \
     chrome_pgo_phase=0 \
-    enable_hangout_services_extension=true
+    enable_hangout_services_extension=true \
+    use_system_libwebp=true
 
 # Closure compile requires Java
 gn_args += \
-- 
2.25.1

